libs/detectors/x86/pedestrian_ssd_mobilenet_v2.py

```diff
@@ -0,0 +1,77 @@
+import pathlib
+import time
+import os
+import numpy as np
+import wget
+import tensorflow as tf
+
+from libs.detectors.utils.fps_calculator import convert_infr_time_to_fps
+
+
+def load_model(model_name):
+    base_url = 'https://raw.githubusercontent.com/neuralet/neuralet-models/master/amd64/'
+    model_file = model_name + "/saved_model/saved_model.pb"
+    base_dir = "libs/detectors/x86/data/"
```

Comment:
Please use `base_dir = "/repo/data/x86"` instead.


I have moved all data files to the top `data` directory so that we can only mount `data` dir, rather than the whole repository, when running the docker container.